iamproperty/iamproperty.github.io
# Ds 62/related cards

## Summary of Changes:
1. Add a new example to show how the related cards component in the media works designs
2. Introduce circle patterns

## Rationale
Demonstrate how to implement a design from media works.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /examples/related-cards

## Ticket(s)
DS-62

## Checklist

## Notes
A change to the container CSS has led to a lot of visual regression changes.
